devnet/substrate_alice.Dockerfile

```
@@ -1,35 +1,48 @@
-FROM parity/polkadot:v0.9.10 AS polkadot
-FROM golang:1.17
+# Copyright 2022 ChainSafe Systems (ON)
+# SPDX-License-Identifier: LGPL-3.0-only
+
+FROM golang:1.17 as openmetrics
+ARG METRICS_NAMESPACE=substrate.local.devnet
+
+WORKDIR /devnet
+
+COPY ./devnet/go.mod ./devnet/go.sum ./
+RUN go mod download
+
+COPY ./devnet .
+RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alice > conf.yaml
+
+FROM parity/polkadot:v0.9.17
 
 ARG CHAIN=cross-client
-ARG VERSION=v0.9.10
 ARG DD_API_KEY=somekey
-ARG METRICS_NAMESPACE=substrate.local.devnet
 
-ENV CHAIN=${CHAIN}
 ENV DD_API_KEY=${DD_API_KEY}
+ENV CHAIN=${CHAIN}
 
-RUN DD_AGENT_MAJOR_VERSION=7 DD_INSTALL_ONLY=true DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+USER root
+RUN apt update && apt install -y curl
```

supernit just to have a smaller image

```suggestion
RUN apt update && apt install -y curl && rm -r /var/cache/* /var/lib/apt/lists/*
```